asreviewcontrib/insights/plot.py

```diff
@@ -52,85 +53,27 @@ def plot_recall(ax,
     return _plot_recall(ax,
                         labels,
                         x_absolute=x_absolute,
-                        y_absolute=y_absolute)
-
-
-def _plot_recall(ax, labels, x_absolute=False, y_absolute=False):
-    """Plot the recall of state object(s).
-
-    labels:
-        An ASReview state object.
-    """
-
-    x, y = _recall_values(labels, x_absolute=x_absolute, y_absolute=y_absolute)
-
-    if y_absolute:
-        n_pos_docs = sum(labels)
-        y_lim = [-n_pos_docs * 0.05, n_pos_docs * 1.05]
-        yticks = [int(n_pos_docs * r) for r in [0, 0.2, 0.4, 0.6, 0.8, 1.0]]
-    else:
-        y_lim = [-0.05, 1.05]
-        yticks = [0, 0.2, 0.4, 0.6, 0.8, 1.0]
-
-    if x_absolute:
-        xlabel = "Number of labeled records"
-    else:
-        xlabel = "Proportion of labeled records"
-
-    ax.step(x, y, where='post')
-    ax.set_title("Recall")
-    ax.set(xlabel=xlabel, ylabel='Recall')
-    ax.set_ylim(y_lim)
-    ax.set_yticks(yticks)
-
-    if x_absolute:
-        ax.xaxis.get_major_locator().set_params(integer=True)
-
-    # add random line if required
-    _plot_random_recall(ax,
-                        labels,
-                        x_absolute=x_absolute,
-                        y_absolute=y_absolute)
-
-    # correct x axis if tick is at position 0
-    _fix_start_tick(ax)
-
-    return ax
-
-
-def _plot_random_recall(ax, labels, x_absolute, y_absolute):
-    """Plot the recall of state object(s).
-
-    labels:
-        An ASReview state object.
-    """
-    n_docs = len(labels)
-    n_pos_docs = sum(labels)
-
-    # add random line if required
-    x = np.arange(1, n_docs + 1)
-    recall_random = np.round(np.linspace(0, n_pos_docs, n_docs))
-
-    if not x_absolute:
-        x = x / n_docs
-
-    if y_absolute:
-        y = recall_random
-    else:
-        y = recall_random / n_pos_docs
-
-    ax.step(x, y, color="black", where='post')
-
-    return ax
-
-
-def plot_wss(ax, state_obj, priors=False, x_absolute=False, y_absolute=False):
+                        y_absolute=y_absolute,
+                        show_random=show_random,
+                        show_legend=show_legend,
+                        legend_values=legend_values)
+
+
+def plot_wss(ax,
+             state_obj,
+             priors=False,
+             x_absolute=False,
+             y_absolute=False,
+             show_legend=True,
+             legend_values=None):
     """Plot the WSS@T for all thresholds T.
 
     Arguments
     ---------
-    state_obj: asreview.state.SQLiteState
-        State object from which to get the labels for the plot.
+    state_obj: Union[asreview.state.SQLiteState,
+                     list[asreview.state.SQLiteState]]
```

Comment:
I think documenting the types is not so common. I think sphinx can handle typing nowadays very well. You can skip this if you apply typing. I would suggest using traditional docstrings and doing proper typing in another PR.